src/test/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbStreamTestRunner.java

```diff
@@ -179,24 +179,24 @@ public void shutdown(ShutdownInput shutdownInput) {
         private final ExecutorService workerPool = Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors() * 2);
         private final Map<String, Worker> workers = new HashMap<>();
 
-        private final AmazonDynamoDB amazonDynamoDB;
-        private final AmazonDynamoDBStreams amazonDynamoDBStreams;
+        private final AmazonDynamoDB amazonDynamoDb;
+        private final AmazonDynamoDBStreams amazonDynamoDbStreams;
         private final AWSCredentialsProvider awsCredentialsProvider;
         private final CountDownLatch countDownLatch;
-        private final List<MTRecord> recordsReceived;
+        private final List<MtAmazonDynamoDb.MtRecord> recordsReceived;
```

Comment:
why is this expanded to explicitly denote the outer class (i.e., "MtAmazonDynamoDb.MtRecord" instead of just "MtRecord")?